strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/configuration/StoragesConfigurationControllerTestIT.java

```diff
@@ -62,143 +65,207 @@ static ProxyConfigurationForm createProxyConfiguration()
         return proxyConfiguration;
     }
 
+    @Override
+    @BeforeEach
+    public void init()
+            throws Exception
+    {
+        super.init();
+        setContextBaseUrl("/api/configuration/strongbox/storages");
+
+        r0 = new RepositoryForm();
```

Comment:
Could you please use `repositoryForm0` as the naming convention? This is easier for refactoring later. I'm only mentioning this, because I had to modify a few such classes when I was working on the test parallelization and I came across such cases where it was a bit trickier to re-work because of inconsistencies in the namings. (Thanks!) :)